website/js/components/VariantSubmitter.js

```diff
@@ -135,15 +136,19 @@ const VariantSubmitter = React.createClass({
         const {submitter, cols, data} = this.props;
 
         // for each panel, construct key-value pairs as a row of the table
-        const submitterRows = cols.map(({prop, title, value, helpKey}) => {
+        const submitterRows = cols.map(({prop, title, value, noHelpLink}) => {
             const isEmptyValue = util.isEmptyField(value);
             const rowItem = util.getFormattedFieldByProp(prop, data);
 
             return (
                 <tr key={prop} className={ (isEmptyValue && this.props.hideEmptyItems) ? "variantfield-empty" : "" }>
                     {
-                        helpKey
-                            ? <KeyInline tableKey={title} onClick={(event) => this.props.showHelp(event, helpKey)}/>
+                        !noHelpLink
```

Comment:
minor, but why the double negative?

Reply:
Heh, yeah, that's definitely odd; nice catch. I've reversed the clauses and removed the double-negative. (For reference the field actually isn't used yet, but I should probably specify it for the fields for which we don't have help text yet...)

Reply:
I should probably also add that functionality to every place we use KeyInline (the main details display component and the allele frequency tile) if I do use it, too.